ambry-rest/src/main/java/com.github.ambry.rest/NettyMessageProcessor.java

```diff
@@ -55,7 +55,7 @@
  * keepalive, requests can follow one after the other. But at any point of time, only one request is actually "alive"
  * in the channel (i.e. there cannot be multiple requests in flight that are being actively served on the same channel).
  */
-class NettyMessageProcessor extends SimpleChannelInboundHandler<HttpObject> {
+public class NettyMessageProcessor extends SimpleChannelInboundHandler<HttpObject> {
```

Comment:
This was made `public` to support different implementations of `NettyServerFactory` (different implementations would be needed if the pipeline had to be changed).